projects/core/src/lib/backend/backend.model.ts

```diff
@@ -548,6 +548,7 @@ export interface LayerCollectionItemDict {
     id: ProviderLayerIdDict | ProviderLayerCollectionIdDict;
     name: string;
     description: string;
+    properties: [string, string][];
```

Comment:
```suggestion
    properties: Array<[string, string]>;
```

would be my preference.
Where is Clippy when we would need it :smile: .

Reply:
65dbbac9aaf94ea02c8d34d5acc86522d619ebd1